### PR TITLE
Fixed Navbar Issue

### LINKED ISSUE #8: Fix Bootstrap Navigation bug so the body content changes according which tab the user has selected.

### PR DESCRIPTION
**Pull request by SivamAgarwalla**
Changes
-Placed the list that renders all the user note URLS on a separate html
-used NavBar component to link to this page
-popup.html page is the initial page shown by the navbar

### CODE REVIEW

File: src/popup.html [hunk 1]
```diff
@@ -7,37 +7,28 @@
...
-          <div id="notes-urls"></div>
-        </div>
-      </div>
+    </nav>
 
+    <div class="container" id="main__add__section">
+      <textarea id="notes-content" class="form-control"></textarea>
+      <button id="save-button" type="button" class="btn-primary">Save</button>
     </div>
+    <div class="alerts"></div>
```
Comment (jontsai) [inline src/popup.html:31]: not really important, but should this go above the main `.container` div?

File: src/js/popup.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-$(function() {
+$(function () {
```
Comment (jontsai) [inline src/js/popup.js:1]: Was this space automatically inserted by VSCode?

Just wondering.

File: src/popup.html [hunk 3]
```diff
@@ -7,37 +7,28 @@
...
-          <button id="save-button" type="button" class="btn-primary">Save</button>
-        </div>
-        <div class="tab-pane fade show active" id="nav-all-notes" role="tabpanel"
-             aria-labelledby="nav-all-notes-tab">
-          <div id="notes-urls"></div>
-        </div>
-      </div>
+    </nav>
 
+    <div class="container" id="main__add__section">
```
Comment (jontsai) [inline src/popup.html:27]: Just curious, what naming convention is this using?

**Review by jontsai — APPROVED** [review]
mostly nitpicky comments on style/naming conventions

rest is LGTM

File: src/js/popup.js (hunk 2 above)
Comment (SivamAgarwalla) [reply src/js/popup.js:1]: Yes I believe so, I use Prettier extension for my VSCode. It was most likely auto-formatting.

File: src/popup.html (hunk 3 above)
Comment (SivamAgarwalla) [reply src/popup.html:27]: This is definitely a mistake, I'll make sure one of goes through our HTML files soon and we can decide on a naming convention and change everything to stick to that. For now it is a little unorganized.

**Review by adot18 — APPROVED** [review]

**Pull request merged**